tests/automation_framework/data/worker/worker_update.ini

```diff
@@ -0,0 +1,24 @@
+[DEFAULT]
+method=WorkerUpdate
+id=2
+params:{
+            "workerId":"",
+            "details":
+            {
+                "hashingAlgorithm": "KECCAK-256"
+            }
+        }
+
+[18266]
+workerId=4582
+hashingAlgorithm=remove
+details:{"name": "TEST"}
+
+[18267]
```

Comment:
Again same like above, the test is test_worker_update_invalid_parameter, why are we removing hashingalgorithm?